test/models/review-result.model.test.js

```diff
@@ -0,0 +1,99 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseHelper = require('../support/helpers/database.helper.js')
+const ReviewChargeElementResultHelper = require('../support/helpers/review-charge-element-result.helper.js')
+const ReviewChargeElementResultModel = require('../../app/models/review-charge-element-result.model.js')
+const ReviewResultHelper = require('../support/helpers/review-result.helper.js')
+const ReviewReturnResultHelper = require('../support/helpers/review-return-result.helper.js')
+const ReviewReturnResultModel = require('../../app/models/review-return-result.model.js')
+
+// Thing under test
+const ReviewResultModel = require('../../app/models/review-result.model.js')
+
+describe('Review Result model', () => {
+  let testRecord
+
+  beforeEach(async () => {
+    await DatabaseHelper.clean()
+  })
+
+  describe('Basic query', () => {
+    beforeEach(async () => {
+      testRecord = await ReviewResultHelper.add()
+    })
+
+    it('can successfully run a basic query', async () => {
+      const result = await ReviewResultModel.query().findById(testRecord.id)
+
+      expect(result).to.be.an.instanceOf(ReviewResultModel)
+      expect(result.id).to.equal(testRecord.id)
+    })
+  })
+
+  describe('Relationships', () => {
+    describe('when linking to review charge element result', () => {
+      let testReviewChargeElementResult
+
+      beforeEach(async () => {
+        testReviewChargeElementResult = await ReviewChargeElementResultHelper.add()
+        testRecord = await ReviewResultHelper.add({ reviewChargeElementResultId: testReviewChargeElementResult.id })
+      })
+
+      it('can successfully run a related query', async () => {
+        const query = await ReviewResultModel.query()
+          .innerJoinRelated('reviewChargeElementResults')
+
+        expect(query).to.exist()
+      })
+
+      it('can eager load the ', async () => {
+        const result = await ReviewResultModel.query()
+          .findById(testRecord.id)
+          .withGraphFetched('reviewChargeElementResults')
+
+        expect(result).to.be.instanceOf(ReviewResultModel)
+        expect(result.id).to.equal(testRecord.id)
+
+        expect(result.reviewChargeElementResults[0]).to.be.instanceOf(ReviewChargeElementResultModel)
+        expect(result.reviewChargeElementResults[0]).to.equal(testReviewChargeElementResult)
+      })
+    })
```

Comment:
If you look at tests for other models we have 2 test patterns we follow; one for `BelongsToOneRelation` and another for `HasManyRelation`.

The key difference is whether we create 1 or multiple test records, and then confirm if we get a single object or array of them back.

This test is the pattern for `BelongsToOneRelation` but the relationship is actually a `HasManyRelation`.

If it helps `test/models/crm-v2/billing-account.model.test.js` has an example of both types that doesn't get too mad!